src/apps/applets/api/applets.py

```diff
@@ -131,33 +129,10 @@ async def applet_create(
         )
         manager_role = Role.EDITOR if has_editor else None
 
-        mail_service = MailingService()
         try:
             applet = await AppletService(session, owner_id).create(schema, user.id, manager_role)
         except Exception:
-            await mail_service.send(
-                MessageSchema(
-                    recipients=[user.email_encrypted],
-                    subject="Applet upload failed!",
-                    body=mail_service.get_template(
-                        path="applet_create_failed_en",
-                        first_name=user.first_name,
-                        applet_name=schema.display_name,
-                    ),
-                )
-            )
             raise
```

Comment:
If error handling is removed, then try|catch is not needed here

Reply:
Oops, good catch!